stdlib/random.mli

```diff
@@ -147,6 +147,29 @@ module State : sig
       correlation.  Both PRNGs can be split later, arbitrarily many times.
       @since 5.0 *)
 
+  val to_octets : t -> string
+  (** Serializes the PRNG state into an immutable octet sequence.
```

Comment:
I always prefer when docstrings have the form `[f v] is ...`, but I suspect I get on people's nerve with that obsession. 

Also I would say that you seralize `to` rather than `into` (which I would rather use for an in place modification) but IANANES.